core/controllers/admin_test.py

```diff
@@ -294,7 +294,7 @@ def test_cancel_one_off_job(self):
         with self.swap(
             jobs_registry, 'ONE_OFF_JOB_MANAGERS', [SampleMapReduceJobManager]):
 
-            self.get_html_response('/admin')
+            self.get_json('/adminhandler')
```

Comment:
wanted to know the reason behind this change?

Reply:
In PR https://github.com/oppia/oppia/pull/7186, some of the functions in AdminPage and AdminHandler were swapped and hence this decreased coverage.